bitnami/common/templates/_labels.tpl

```diff
@@ -3,7 +3,12 @@
 Kubernetes standard labels
 */}}
 {{- define "common.labels.standard" -}}
+app: {{ include "common.names.name" . }}
```

Comment:
We already included this label for certain charts to be compatible with Istio (see https://github.com/bitnami/charts/pull/2286)
That said, I'd like Istio to move forward to embrace the [default k8s recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/). `app`, `version`, etc. labels are not recommended by k8s since quite a lot time.
I have mixed feelings about adding this since we're duplicating labels.

Reply:
My 2cts is to follow the k8s recommendations as much as possible, if there is a specific asset that needs to follow a different approach due to a widely common use case we can evaluate an exception for this kind of case. But adding non-recommended and duplicated labels to the different k8s objects in this library chart that is going to be applied to the whole catalog doesn't sound like the best option to me.

Do you know why Istio is not using the recommended labels (any open issue or similar)? I'm just thinking out loud, but I am not sure if we should create this (potentially risky) workaround to support something it's not recommended by the core maintainers.

Reply:
It looks like Istio is moving more towards the recommended Kubernetes labels so this might not be as needed as I first though ref. https://github.com/istio/istio/issues/10724

Reply:
I have removed the legacy labels.